website/content/docs/k8s/helm.mdx

```diff
@@ -169,16 +169,35 @@ Use these links to navigate to a particular top-level stanza.
         - gossip encryption key defined by `global.gossipEncryption.secretName`.
         To discover the service account name of the Consul client, run
             ```shell-session
-            $ helm template --show-only templates/client-serviceaccount.yaml <release-name> charts/consul
+            $ helm template --show-only templates/client-serviceaccount.yaml <release-name> hashicorp/consul
+            ```
+        and check the name of `metadata.name`.
+
+      - `consulSnapshotAgentRole` ((#v-global-secretsbackend-vault-consulsnapshotagentrole)) (`string: ""`) - <EnterpriseAlert inline /> The Vault role for the Consul client snapshot agent.
+        The role must be connected to the Consul client snapshot agent's service account and
+        have a policy with read capabilities for the snapshot agent config defined by `client.snapshotAgent.configSecret.secretName`.
+        To discover the service account name of the Consul client, run
+            ```shell-session
+            $ helm template --show-only templates/client-snapshot-agent-serviceaccount.yaml --set client.snapshotAgent.enabled=true <release-name> hashicorp/consul
             ```
         and check the name of `metadata.name`.
 
       - `manageSystemACLsRole` ((#v-global-secretsbackend-vault-managesystemaclsrole)) (`string: ""`) - A Vault role to allow Kubernetes job that manages ACLs for this Helm chart (`server-acl-init`)
-        to read and update Vault secrets for the Consul's bootstrap and replication tokens.
+        to read and update Vault secrets for the Consul's bootstrap, replication or partition tokens.
         This role must be bound the `server-acl-init`'s service account.
         To discover the service account name of the `server-acl-init` job, run
             ```shell-session
-            $ helm template --show-only templates/server-acl-init-serviceaccount.yaml <release-name> charts/consul
+            $ helm template --show-only templates/server-acl-init-serviceaccount.yaml \
+                  --set global.acls.manageSystemACLs=true <release-name> hashicorp/consul
+            ```
+        and check the name of `metadata.name`.
+
+      - `adminPartitionsRole` ((#v-global-secretsbackend-vault-adminpartitionsrole)) (`string: ""`) - <EnterpriseAlert inline /> A Vault role to allow Kubernetes job that creates a Consul partition for this Helm chart (`partition-init`)
+        to read Vault secret for the partition ACL token.
+        This role must be bound the `partition-init`'s service account.
+        To discover the service account name of the `partition-init` job, run with Helm values for the client cluster:
```

Comment:
Let's revisit this paragraph. I'm not sure I understand what this role is doing.  It seems like it's used for a Kubernetes job that creates the admin partition.  But then the next sentence confuses me "to read Vault secret for the partition ACL token." Is this part of the first sentence or a new sentence?